multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -174,10 +176,9 @@ write_files:
 
       sudo rkt run \
         --net=host \
-        --volume bk,kind=host,source=/srv/kubernetes \
-        --mount volume=bk,target=/srv/kubernetes \
         --uuid-file-save="$uuid_file_save" \
         --trust-keys-from-https \
+        ${RKT_OPTS:-} \
```

Comment:
figured it works the same was `kubectl-wrapper` in this regard. Both are wrapping `rkt run hyperkube`